flake8_quotes/__init__.py

```diff
@@ -54,10 +54,12 @@ def _register_opt(parser, *args, **kwargs):
     @classmethod
     def add_options(cls, parser):
         cls._register_opt(parser, '--quotes', action='store',
-                          parse_from_config=True,
+                          parse_from_config=True, type='choice',
+                          choices=list(cls.INLINE_QUOTES),
```

Comment:
To be honest if someone doesn't know if it uses keys, values or both, they probably don't know what it actually does even if there is `.keys()`? I mean everybody probably at some point did iterate over a dict and only got the keys.

But sure it could explicitly call `.keys()`  and maybe even do a sorted thing.